core/state_processor.go

```diff
@@ -144,7 +144,7 @@ func applyTransaction(msg *Message, config *params.ChainConfig, gp *GasPool, sta
 	// Apply the transaction to the current state (included in the env).
 	applyMessageStartTime := time.Now()
 	result, err := ApplyMessage(evm, msg, gp, l1DataFee)
-	if evm.Config.Tracer != nil && evm.Config.Tracer.IsDebug() {
+	if evm.Config.Tracer != nil {
```

Comment:
Will this open a counter-based ccc tracer for all nodes?

Reply:
It was already enabled on all nodes if I am not mistaken. This change doesn't really affect that.